Octokit/Models/Response/Readme.cs

```diff
@@ -30,6 +30,22 @@ internal Readme(ReadmeResponse response, IApiConnection client)
             htmlContent = new Lazy<Task<string>>(async () => await client.GetHtml(Url).ConfigureAwait(false));
         }
 
+        internal Readme(ReadmeResponse response, IConnection client)
```

Comment:
What's the reasoning behind adding this additional constructor?  I notice it's only used in the Observable tests, but couldnt they just pass in the IApiConnection and use the existing constructor, rather than passing in an IConnection and having an entire new constructor (and mostly duplicated logic)?  Apologies if i've missed something, as im on my phone :grinning: